src/topology/algebra/weak_dual_topology.lean

```diff
@@ -0,0 +1,87 @@
+/-
+Copyright (c) 2021 Kalle Kytölä. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kalle Kytölä and Heather Macbeth
+-/
+import tactic
```

Comment:
You don't need this import, in fact (by the time you get to files about functional analysis, most/all tactics have been imported already).

Reply:
Thanks! The linter was also not happy about this. I tend to `import tactic` in my own playing around, but will try to remember to remove it from PRs in the future.